component/core/src/test/java/org/exoplatform/social/core/utils/MentionUtilsTest.java

```diff
@@ -0,0 +1,67 @@
+package org.exoplatform.social.core.utils;
+
+import org.exoplatform.social.core.identity.model.Identity;
+import org.exoplatform.social.core.identity.model.Profile;
+import org.exoplatform.social.core.identity.provider.OrganizationIdentityProvider;
+import org.exoplatform.social.core.manager.IdentityManager;
+import org.exoplatform.social.core.test.AbstractCoreTest;
+import org.junit.Test;
+
+import java.util.regex.Pattern;
+
+
+public class MentionUtilsTest extends AbstractCoreTest {
+
+    private IdentityManager identityManager;
+
+    private static final Pattern MENTION_PATTERN                 = Pattern.compile("@([^\\s<]+)|@([^\\s<]+)$");
+
+    public void setUp() throws Exception {
+        super.setUp();
+        System.setProperty("gatein.email.domain.url", "http://exoplatfom.com");
+        identityManager = getContainer().getComponentInstanceOfType(IdentityManager.class);
+
+    }
+
+    @Test
+    public void test_substituteUsernames(){
```

Comment:
Please use camel case for function names

Reply:
done